__tests__/app.test.js

```diff
@@ -159,6 +159,23 @@ describe("GET", () => {
         });
     });
   });
+  describe('/api/users', () => {
+    test('STATUS 200, should respond with an array of objects', () => {
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+          const { users } = response.body;
+          users.map((user) => {
```

Comment:
You should be using a forEach here as were don't need to return anything into a new array which is what map is designed to do. We should also check the length of the returned array is correct when using the forEach to protect against a false positive from an accidental empty array being returned and the forEach not running

Reply:
I know!
 I don't know why I went with map probably because it has only 3 letters  ; )
Will check the length. Missed it for one of the previous tests too!
Thanks for the review.